src/ext/java/org/opentripplanner/ext/vectortiles/layers/stops/StopsBaseLayerBuilder.java

```diff
@@ -0,0 +1,30 @@
+package org.opentripplanner.ext.vectortiles.layers.stops;
+
+import static java.util.Map.entry;
+
+import java.util.Locale;
+import java.util.Map;
+import org.opentripplanner.ext.vectortiles.VectorTilesResource;
+import org.opentripplanner.inspector.vector.LayerParameters;
+import org.opentripplanner.transit.service.TransitService;
+
+public class StopsBaseLayerBuilder extends StopsLayerBuilder {
```

Comment:
Is this class necessary? Could we just edit the existing StopsLayerBuilder since this is not really overriding anything.